aws/eks/eks-how-to/build-ubuntu-pro-cluster.rst

```diff
@@ -0,0 +1,103 @@
+Build an Ubuntu Pro K8s cluster
+===============================
+
```

Comment:
please introduce what is being done. it feels weird for a reader to start a k8s deployment guide with something like Packer...why? what is being done?